release/models/gnsi/openconfig-gnsi-certz.yang

```diff
@@ -183,9 +189,6 @@ module openconfig-gnsi-certz {
         "The timestamp of the moment when the authentication policy
         that is currently used by this gRPC server was created.";
     }
-    // TODO(earies): Consider aligning this with grpc-server key after moving
-    // to a system-wide subtree:
-    // https://github.com/openconfig/public/issues/1050
     leaf ssl-profile-id {
```

Comment:
For consistency, we'd also have to update the name of fields in the gNSI Certz model (grep for `ssl` in https://github.com/openconfig/gnsi/blob/0342fdddb70889c9a9bfb87134e5ca0272c70c86/certz/certz.proto#L292),

as well as in the certz docs: https://github.com/openconfig/gnsi/blob/0342fdddb70889c9a9bfb87134e5ca0272c70c86/certz/README.md#ssl-profiles

and the references to ssl profiles in 
https://github.com/openconfig/attestz/blob/31e1654a0ed0c9fcd00fb7c2165d0f5c4811eea7/README.md
and
https://github.com/openconfig/attestz/blob/31e1654a0ed0c9fcd00fb7c2165d0f5c4811eea7/proto/tpm_enrollz.proto#L62

(there may be other references which i have missed, as the concept of an "SSL profile" in Certz has been around since dec 2022 https://github.com/openconfig/gnsi/commit/d290404c2645ec0cf19448830ee7a5cb2e70438f) 

I feel it is out of the scope of this change to modify this certz leaf, but i'll leave it to @marcushines to comment

Reply:
i think we should keep ssl-profile-id as the consistent name until we can change all locations of it to a more proper name of tls-profile-id